tests/openwisp2/settings.py

```diff
@@ -86,6 +88,7 @@
 ]
 
 MIDDLEWARE = [
+    # 'debug_toolbar.middleware.DebugToolbarMiddleware',
```

Comment:
seems you forgot some debugging lines in this file

Reply:
Yeah, sorry for causing the confusion again, when I mentioned on gitter, it wasn't ready for review! :smile: